Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/resources/processdiscoverer/zul/components/logStats.zul

```diff
@@ -80,7 +80,7 @@
       <vlayout spacing="0">
         <n:div class="ap-pd-logstats-header ap-link" title="Filter by Events">
           <button id="btnEventHeading" sclass="ap-icon ap-icon-flag" ></button>
-          <label id="lblEventHeading" sclass="title" value="Events"/>
+          <label id="lblEventHeading" sclass="title" value="Activity inst."/>
```

Comment:
From testing with logs having both start and complete events, it shows the total events in the log, not 'activity instance'.

![image](https://user-images.githubusercontent.com/12888841/117394244-27c2b300-af39-11eb-8af5-50c8dc136322.png)

Reply:
Confirmed with Marcello:

![image](https://user-images.githubusercontent.com/12888841/117395246-e9c68e80-af3a-11eb-9561-e12c615130ac.png)

Reply:
Inconsistent UI: both "Event" and "Activity Instance" are used

![image](https://user-images.githubusercontent.com/12888841/117397355-4a57ca80-af3f-11eb-9e25-5a9a8f56f46f.png)